projects/components/src/expander/expander-toggle.component.ts

```diff
@@ -27,6 +29,9 @@ export class ExpanderToggleComponent {
   @Input()
   public showTooltip: boolean = true;
 
+  @Input()
+  public colorChange: boolean = false;
```

Comment:
Do we need this as an input? Are there any place where we would want to use the old behavior?

Reply:
I added to make it more customizable. Do you think that I should remove it? We're are going to use it with the change in all ocurrences.

Reply:
If we don't need it now, I'd remove it - it's much easier to add a new input later when we have a use case than remove it.

As a separate comment (not relevant if removing it) - the suffix `*Change` is generally reserved for outputs, as that's the angular framework's naming expectation for banana in box syntax (https://angular.io/guide/two-way-binding#how-two-way-binding-works)

Reply:
great, thanks for the advice I appreciate it

Reply:
This got removed in one commit, then added back in the next - was that intentional?

Reply:
It should need to be removed, I added it by mistake.